apps/fulfillment/index.html

```diff
@@ -3,7 +3,6 @@
   <head>
     <meta charset="utf-8" />
     <title>Fulfillment App</title>
-    <base href="/" />
 
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="description" content="Fulfillment App" />
```

Comment:
let's drop description as it will be provided by page meta data

Reply:
I also see:

```html
<meta name="msapplication-TileColor" content="#ffffff" />
<meta name="theme-color" content="#ffffff" />
```

Can we get ride of those? For now, we can simply remove them, going forward,  I can imagine we add them through themes.

Reply:
Can we drop the favicon as well, and the latest icon? We can integrate favicon through meta service as we do with storefront

Reply:
The goal is to have the same index.html as we have with storefront.

Reply:
`description`, `theme-color`, `favicon`, `mask-icon`, `apple-touch-icon` are required for minimal PWA setup. In case we move it to `meta` we can have issues with Lighthouse scoring and others. In some configurations PWA build is not working at all.

We can drop `msapplication-TileColor` as it's required for Windows only :)

Reply:
Statement: any non-ssr sites should not bother with Lighthouse. 
Can you reconsider in the _light_ of this statement.